### ISSUE TITLE
Not all content read and displayed properly and incorrect focus location with TalkBack on

### ISSUE THREAD

**Issue by qanitah** [cds-snc/covid-shield-mobile#535]
When user goes to "Learn more about how it works" from onboarding screens, they are taken to "How COVID Alert works" carousel. There are a couple of issues with this with TalkBack on.

1. When "Read from top" from Global menu in the first screen (or swiping right item by item), all 4 screens are read one after another and Close and bottom is not read. It doesn't tell the user they're on which screen of 4. The bottom does not change. 

2. Since the bottom does not change, when the user is on the last screen, "Next" is still displayed and double tapping on it take the user to the following screen from where they started reading "Read from top" from (or swiping right item by item). They have to go through remaining screens again. So if they start from #1, it'll go to screen #2 after the last, then go to screen #3 and so on. 

3. When user taps on "Next" the focus stays on "Next". Same for "Back" button and stays on "Back".

Android 8.1.0 on Samsung Galaxy
Android 10 on OnePlus 7

![image](https://user-images.githubusercontent.com/14280062/86943157-17248f00-c114-11ea-8471-46e9564bde06.png)

![image](https://user-images.githubusercontent.com/14280062/86943484-81d5ca80-c114-11ea-9bc6-f7816b4e3cbf.png)

![image](https://user-images.githubusercontent.com/14280062/86943507-869a7e80-c114-11ea-9293-5e9b2148664c.png)

**Comment by nickmly** [comment]
Cannot re-create using latest version. I believe this was fixed in #496

**Comment by JuliannaR** [comment]
@qanitah this is fixed in the most recent update that came out just after noon today.
Here is a video of the experience in talkback from V28. https://drive.google.com/file/d/1mbD3j6im_zOwscpC0loeq5tqFp0qsUEw/view?usp=sharing

**Comment by qanitah** [comment]
Yes these issues are all fixed. Closing this issue..
